scylla-gdb.py

```diff
@@ -4174,13 +4196,22 @@ class scylla_find(gdb.Command):
       thread 1, small (size <= 512), live (0x6000000f3800 +48)
       thread 1, small (size <= 56), live (0x6000008a1230 +32)
     """
-    _vptr_type = gdb.lookup_type('uintptr_t').pointer()
-    _size_char_to_size = {
-        'b': 8,
-        'h': 16,
-        'w': 32,
-        'g': 64,
-    }
+    @classmethod
+    @property
+    @functools.cache
+    def _vptr_type(cls):
+        return gdb.lookup_type('uintptr_t').pointer()
+
+    @classmethod
+    @property
+    @functools.cache
+    def _size_char_to_size(cls):
+        return {
+            'b': 8,
+            'h': 16,
+            'w': 32,
+            'g': 64,
+        }
```

Comment:
Why convert _size_char_to_size? It's a true constant.

Reply:
it represents a mapping, which takes a key and returns value. so i turned it into a function. please see the commit message of [cd2229b](https://github.com/scylladb/scylladb/pull/20334/commits/cd2229bb6f3242874eed6ba2f6f6d6fc8043e0fd) . i can drop this commit if this change does not make sense.

Reply:
Changing it to a function makes sense, but why a cached function? The cache will slap a dict on top of the dict it's implemented with. It doesn't really matter, but looks strange. Also the initial conversion to a cached property doesn't make sense to me.

Reply:
ahh, yeah. i was doing some semi-mechanic editing. will use a plain property and then a plain function in the next revision.